patterns/nvidia-gpu-efa/eks.tf

```diff
@@ -35,6 +35,15 @@ module "eks" {
       ami_type       = "AL2_x86_64_GPU"
       instance_types = ["p5.48xlarge"]
 
+
+      # Uncomment this block and specify capacity_reservation_id
```

Comment:
remove this - we have a dedicated example that shows the recommended usage https://github.com/aws-ia/terraform-aws-eks-blueprints/tree/main/patterns/targeted-odcr

Reply:
removed as requested